packages/dds/tree/src/simple-tree/core/walkSchema.ts

```diff
@@ -69,4 +71,9 @@ export interface SchemaVisitor {
 	 * This includes every field, but also the allowed types array for maps and arrays and the root if starting at {@link walkAllowedTypes}.
 	 */
 	allowedTypes?: (allowedTypes: NormalizedAnnotatedAllowedTypes) => void;
+	/**
+	 * If true, will walk the staged allowed types of the schema in both the node callback and the allowedTypes callback.
+	 * If undefined, will skip any staged allowed types in the node callback but will include them in the allowedTypes callback.
+	 */
+	walkStagedAllowedTypes?: true;
```

Comment:
This isn't actually used by anything but I feel like it's a nice option to have, can remove if you think it's unnecessary

Reply:
I think we need to use this when constructing the stored schema used in the default unhydrated contexts. If we don't, our import and node construction APIs will fail to construct trees which use staged allowed types. This would make operations like exportVerbose then importVerbose fail to clone trees using typos which are staged in the current app code, but enabled in the document.

We should make sure such cases have tests